pack/side/mtt_encounter.json

```diff
@@ -63,5 +604,476 @@
+        "back_traits": "Machination.",
+        "code": "87035",
+        "double_sided": true,
+        "encounter_code": "machinations_through_time",
+        "encounter_position": 34,
+        "faction_code": "mythos",
+        "flavor": "Nothing in the past appears to be out of the ordinary. You wonder if Edwin Bennet is trustworthy. Surely his vast knowledge could prove extremely useful in aiding you. Reluctantly, you convince yourself the payoff is worth the risk, but it will take a great deal of persuasion to keep Edwin from betraying you.",
+        "name": "Uneasy Alliance",
+        "pack_code": "mtt",
+        "position": 35,
+        "quantity": 1,
+        "text": "<b>Setup</b>: Follow the instructions for your relevant era(s), below. Then, flip this story card over.<hr><b>Past</b>: Place Thomas Corrigan at Childhood Home. Place Mary Zielinski at O'Malley's Watch Shop. Place the Edwin Bennet asset at Arkham Gazette with 12 clues on him, then remove 3 [per_investigator] clues from him.<hr><b>Present and Future</b>: Set Thomas Corrigan and Mary Zielinsky aside, out of play: they have been abducted. If you are playing in <i>Epic Multiplayer Mode</i>, set the Edwin Bennet asset aside, out of play.",
+        "traits": "Machination.",
+        "type_code": "story",
+        "victory": 1
+    },
+    {
+        "back_link": "87036b",
+        "code": "87036a",
+        "encounter_code": "machinations_through_time_single_group",
+        "encounter_position": 2,
+        "enemy_damage": 1,
+        "enemy_evade": 2,
+        "enemy_fight": 2,
+        "enemy_horror": 1,
+        "faction_code": "mythos",
+        "illustrator": "Drazenka Kimpel",
+        "is_unique": true,
+        "health": null,
+        "name": "Edwin Bennet",
+        "pack_code": "mtt",
+        "position": 36,
+        "quantity": 1,
+        "subname": "Bitter Adversary",
+        "text": "Patrol (nearest location with a [[Scientist]] asset).\nEdwin Bennet cannot be damaged.\nClues cannot be discovered from Edwin Bennet's location.\n<b>Forced</b> - At the end of the mythos phase: Deal 1 damage to a [[Scientist]] asset at Edwin Bennet's location.",
+        "traits": "Humanoid. Elite.",
+        "type_code": "enemy"
+    },
+    {
+        "code": "87036b",
+        "cost": null,
+        "encounter_code": "machinations_through_time_single_group",
+        "encounter_position": 2,
+        "faction_code": "neutral",
+        "flavor": "A truly brilliant, yet unstable, mastermind.",
+        "health": 2,
+        "illustrator": "Drazenka Kimpel",
```

Comment:
Please add `"hidden": true,`, which we add for all 'back' cards when they are split over two entries.